.github/actions/opa/action.yml

```diff
@@ -0,0 +1,39 @@
+name: 'OPA CI'
```

Comment:
Why change workflow name?
```suggestion
Test OPA Policies
```

Reply:
This is mainly because it's not just testing; it involves fmt check, bundle building, test, opa check, and lint with Regal. So, I thought `CI` better describes the steps.

But I could change it to `Test OPA Policies`.